_fpki/5_fpki_tools.md

```diff
@@ -36,3 +38,4 @@ crt.sh | [External](https://crt.sh/){:target="_blank"}{:rel="noopener noreferrer
 OpenSC | [Github (External)](https://github.com/OpenSC/OpenSC/wiki){:target="_blank"}{:rel="noopener noreferrer"} | OpenSC provides a set of libraries and utilities to work with smart cards. Use OpenSC versions of 0.20.0 or greater to reduce website authentication errors related to TLS 1.3. | System Engineers
 NIST 85B (800-73-4) Test Tool | [NIST (External)](https://csrc.nist.gov/projects/nist-personal-identity-verification-program/software-downloads){:target="_blank"}{:rel="noopener noreferrer"} | Used to pull deep PIV contents when integrating PIV with various infrastructure components. | System Engineers; FPKI Card Issuers
 Treasury PIV Authentication Tester (TLS 1.3) | [External](https://caia-dev.treasury.gov/pagination-spa){:target="_blank"}{:rel="noopener noreferrer"} | A tool from the U.S. Department of the Treasury, Bureau of the Fiscal Service used to test PIV authentication. Click on "log in" and follow the prompts for PIV. This tool will identify if your agency is impacted by TLS 1.3 break and inspect for Trusted Internet Connection (TIC). | System Engineers; PIV Holders
+
```

Comment:
The easiest way to add something is to the table. You should be part of the FICAM Coders team and can create branches in the repo.

Reply:
👍

Reply:
@idmken This was my first thought and set up for this, but the target result kept moving, the result was me trying to add a link to the nav as request... I will circle back around on this.